### ISSUE TITLE
PMP on Windows is having a major UI glitch

### ISSUE THREAD

**Issue by flying-sausages** [plexinc/plex-media-player#991]
FYI the log instructions do not have any new logs since September last year. I've been using PMP even after this, obviously.

### Test environment

**PMS Version:** 1.20.3.3421-54b6523bd (Ubuntu 20.04 server)
**PMP Version:** 1.20.0.1406-f47a81b7
**Platform (Windows/macOS/Embedded RPi/Embedded Intel):** Windows 10

### Steps to reproduce
1. Open app
1. Hover over cover, or observe the On Deck cards
1. Try play video

### Current behavior
- Covers flip on mouse hover
- Video fails to play back

### Expected behavior
- Covers not flipping
- Videos playing

![image](https://user-images.githubusercontent.com/23618693/95784277-d3719080-0cd3-11eb-94e8-8c5b0ee0f999.png)

![ezgif com-video-to-gif](https://user-images.githubusercontent.com/23618693/95784430-16336880-0cd4-11eb-83b3-7de184bceacd.gif)

This persists after re-install. I retrieve my PMP install from Chocolatey

**Comment by flying-sausages** [comment]
More flippage 
![image](https://user-images.githubusercontent.com/23618693/95784730-ad98bb80-0cd4-11eb-89ed-91909a0d8025.png)

**Comment by flying-sausages** [comment]
Also, the broken video is a consistent error code `4294967280`

**Comment by mseeley** [comment]
There's a lot to unpack here. The first step is to resolve which app you're running and grab a new version. Please share a screenshot of the Settings > General screen it should include an application version.

- 1.20.0.1406-f47a81b7 is a Plex for Windows version number. This would explain why you're not seeing any logs for Plex Media Player. I don't expect logs to be helpful here.
- Plex for Window's Chocolatey is here: https://chocolatey.org/packages/plex, PMP is here: https://chocolatey.org/packages/plexmediaplayer. The version numbers align with the latest official downloads: https://www.plex.tv/media-server-downloads/?lang=#plex-app

Please replace all Plex for Windows / PMP versions from Chocolatey with the official downloads. We've never seen this issue before and would like to rule out the Chocolatey.

**Comment by flying-sausages** [comment]
I've already tested this with the official download from your site and it was reproducing.However the issue stopped reproducing itself and it's the weirdest thing ever.

I have WSL2 installed on my ARM machine, tried to play with some VBox machines, and had to disable `hypervisorlaunchtype` in `bcdedit.exe`. So when I set it back to `bcdedit /set hypervisorlaunchtype auto` it all goes back to normal after restart.

I'm absolutely clueless how this is in any way related.
Something something Hyper-v, something something Windows. Hope there's a slight chance this helps.
